common/common.cpp

```diff
@@ -940,6 +940,21 @@ struct common_init_result common_init_from_params(common_params & params) {
         params.sampling.ignore_eos = false;
     }
 
+    if (params.sampling.ignore_eos) {
+        LOG_INF("%s: added EOS logit bias = %f\n", __func__, -INFINITY);
+        params.sampling.logit_bias.push_back({llama_token_eos(model), -INFINITY});
+    }
```

Comment:
Noticed this doesn't really work for models with multiple eog tokens.
```suggestion
    if (params.sampling.ignore_eos) {
        for (llama_token i = 0; i < llama_n_vocab(model); i++) {
            if (llama_token_is_eog(model, i)) {
                LOG_INF("%s: added %s logit bias = %f\n", __func__, common_token_to_piece(lctx, i).c_str(), -INFINITY);
                params.sampling.logit_bias.push_back({i, -INFINITY});
            }
        }
    }
```